### PR TITLE
Read NAME from config as a fallback for notification

### PR DESCRIPTION
**Pull request by LovecraftianHorror**
# Description

## Contributions

- Fixed a thing that I broke
  - Discord notifications first try to read from $NAME and fall back to reading from the config on failure

## Checklist

- [ ] I added one or multiple labels which best describes this PR.
- [ ] I have tested the changes locally.
- [ ] This PR has a reviewer on it. 
- [ ] I have validated my changes in a docker container and on Ubuntu. (Only needed for Odin or Docker Changes)

### CODE REVIEW

File: src/notifications/discord.rs [hunk 1]
```diff
@@ -46,9 +49,20 @@ pub struct DiscordWebHookBody {
 
 impl DiscordWebHookBody {
   pub fn new(event: &NotificationMessage) -> Self {
-    let config_file = config_file();
-    let config = read_config(config_file);
-    let server_name = config.name;
+    // Some contexts currently don't get passed in $NAME so fall back to reading from the config
+    // if it's missing or invalid UTF-8
+    let server_name = match env::var("NAME") {
```
Comment (mbround18) [inline src/notifications/discord.rs:54]:
```rust
let server_name = fetch_var("NAME", config.name);
```

**Review by mbround18 — CHANGES_REQUESTED** [review]
Lets simplify this to clean it up :)

File: src/notifications/discord.rs (hunk 1 above)
Comment (LovecraftianHorror) [reply src/notifications/discord.rs:54]: Won't trying to read from a config file fail on CI though?

Comment (mbround18) [reply src/notifications/discord.rs:54]: you right :P my brain is a bit frazzled from work :P

**Review by mbround18 — APPROVED** [review]

**Pull request merged**